userspace/libsinsp/utils.cpp

```diff
@@ -1896,3 +1896,11 @@ unsigned int read_num_possible_cpus(void)
 
 	return possible_cpus;
 }
+
+///////////////////////////////////////////////////////////////////////////////
+// Log helper
+///////////////////////////////////////////////////////////////////////////////
+void sinsp_scap_debug_log_fn(const char* msg)
```

Comment:
I love the callback :cupid: just thinking, why don't we pass also the verbosity? probably we would have to move it in scap

Reply:
At this point scap doesn't have a concept of logging or verbosity, so IMHO let's do the bare minimum and come up with fancy logging later (we'll want the log levels to be unified between scap and sinsp, at least)

Reply:
yeah i would share the log levels between the 2 libraries, probably we need to define them in scap and use them in sinsp, but we can do that in another PR :)